src/main/java/VASSAL/build/module/PieceWindow.java

```diff
@@ -70,7 +70,7 @@
   public static final String HIDDEN = "hidden"; //$NON-NLS-1$
   protected static UniqueIdManager idMgr = new UniqueIdManager("PieceWindow"); //$NON-NLS-1$
   protected JComponent root;
-  protected ComponentSplitter.SplitPane mainWindowDock;
+  protected SplitPane mainWindowDock;
```

Comment:
This is going to break every module which touches the splitter. It will cause a problem with exactly the same modules as the change I had to revert in 3.3.1.

Reply:
I see I have changed too much, I will propose a version that retains API compatibility.